phpBB/phpbb/cron/task/core/tidy_search.php

```diff
@@ -26,6 +26,7 @@ class tidy_search extends \phpbb\cron\task\base
 	protected $config;
 	protected $db;
 	protected $user;
+	protected $phpbb_dispatcher;
```

Comment:
Could you maybe add a doc block for this? Even though the old ones don't have that, I like it if we at least add a doc block for newly added lines of code.